### PR TITLE
Syndicate Names

### PR DESCRIPTION
**Pull request by Pangogie**
I got really tired of "Operator #"

Appends phonetic and Greek alphabet names to Syndies.

:cl: Pancake
- tweak: The Syndicate have gotten more creative with their names.

### CODE REVIEW

File: Resources/Prototypes/Datasets/Names/syndicate.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+- type: dataset
+  id: syndicate_names_normal
```
Comment (metalgearsloth) [inline Resources/Prototypes/Datasets/Names/syndicate.yml:2]: PascalCase please

File: Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs [hunk 2]
```diff
@@ -157,15 +158,15 @@ private void OnPlayersSpawning(RulePlayerSpawningEvent ev)
...
                     gear = commanderGear;
                     break;
                 case 1:
-                    name = $"Operator #{i}";
+                    name = $"Agent " + _random.Pick(_prototypeManager.Index<DatasetPrototype>("syndicate_names_elite").Values);
                     gear = medicGear;
                     break;
                 default:
-                    name = $"Operator #{i}";
+                    name = $"Operator " + _random.Pick(_prototypeManager.Index<DatasetPrototype>("syndicate_names_normal").Values); ;
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs:169]: You should probably use lists + take and pick because you might get duplicates.

Comment (Pangogie) [reply Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs]: Think I did this the way you wanted.

**Comment by Pangogie** [comment]
Should be good now. I think.

**Pull request merged**